src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -539,7 +557,7 @@ public Number parse(String text, ParsePosition status) {
      * @return the least double value greater than {@code d}
      * @see #previousDouble
      */
-    public static final double nextDouble (double d) {
+    public static double nextDouble (double d) {
```

Comment:
Is removing `final` OK here? Would this allow defining the static method in the subclass?

Reply:
Right, we don't want those methods to now have the ability to be hidden. Got carried away with the IDE suggested tips. Reverted here and in the other occurrence, thanks for correcting.